experiments/chaos/openebs_single_replica_failure/run_litmus_test.yml

```diff
@@ -28,5 +28,8 @@ spec:
           - name: APP_LABEL
             value: 'app=cassandra'
 
+          - name: STG_ENGINE
```

Comment:
Is this test is verified with both jiva and jiva csi as it involves the changes in env

Reply:
yes, this is verified with both, as this env is newly added and with default value as jiva and not jiva-csi, testing of the old way of provisioning jiva volumes is not affected.